src/stan-users-guide/simple_linear_regression/regression_naive.stan

```diff
@@ -0,0 +1,20 @@
+data {
```

Comment:
Same comments as in other model.  I'm not convinced we need a model with a loop as a baseline, but that's a judgement call.